src/patternfly/demos/Page/page-template-header-tools-elements.hbs

```diff
@@ -1,22 +1,22 @@
 {{#> page-header-tools}}
   {{#> page-header-tools-group page-header-tools-group--modifier="pf-m-icons"}}
-    {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Alerts"'}}
+    {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Unread notifications"'}}
+      {{#> notification-badge notification-badge--modifier="pf-m-unread"}}
         <i class="fas fa-bell" aria-hidden="true"></i>
+      {{/notification-badge}}
     {{/button}}
-    {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Settings"'}}
+    {{#> button button--modifier="pf-m-plain pf-m-desktop" button--attribute='aria-label="Settings"'}}
         <i class="fas fa-cog" aria-hidden="true"></i>
     {{/button}}
+    {{#> dropdown id=(concat page--id '-dropdown-kebab-right-aligned-1') dropdown--IsActionMenu="true" dropdown--modifier="pf-m-mobile" dropdown-menu--modifier="pf-m-align-right" dropdown-toggle--modifier="pf-m-plain" dropdown--HasKebabIcon="true" aria-label="Actions"}}
+    {{/dropdown}}
   {{/page-header-tools-group}}
   {{#> page-header-tools-group}}
-    {{#> dropdown id=(concat page--id '-dropdown-kebab-right-aligned-1') dropdown--IsActionMenu="true" dropdown-menu--modifier="pf-m-align-right" dropdown-toggle--modifier="pf-m-mobile pf-m-plain" dropdown--HasKebabIcon="true" aria-label="Actions"}}
-    {{/dropdown}}
-    <div class="pf-m-user">
-      {{#> dropdown id=(concat page--id '-dropdown') dropdown-toggle--modifier="pf-m-plain" dropdown--HasToggleIcon="true"}}
+      {{#> dropdown id=(concat page--id '-dropdown') dropdown-toggle--modifier="pf-m-plain pf-m-desktop" dropdown--HasToggleIcon="true"}}
```

Comment:
you would want to apply the desktop class to `dropdown--modifier`, not `dropdown-toggle--modifier`